target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-e2600ac-c2.dts

```diff
@@ -137,3 +145,35 @@
 	nvmem-cells = <&precal_art_5000>;
 	qcom,ath10k-calibration-variant = "Qxwlan-E2600AC-C2";
 };
+
+&gmac {
+	status = "okay";
+};
+
+&switch {
+	status = "okay";
+};
+
+&swport2 {
+	status = "okay";
+	label = "lan1";
+
+	nvmem-cell-names = "mac-address";
+	nvmem-cells = <&macaddr_gmac0>;
+};
+
+&swport4 {
+	status = "okay";
+	label = "lan2";
+
+	nvmem-cell-names = "mac-address";
+	nvmem-cells = <&macaddr_gmac0>;
+};
+
+&swport5 {
+	status = "okay";
+
+	label = "eth1";
```

Comment:
Same question, why `eth1`?

Reply:
ok,for ease of use,so it is eth1

Reply:
Is there anything printed on the board or case with port names?

Cause, if not then `eth1` makes no sense, either make it `wan` or `lan2`

Reply:
yes, There is a silk screen on the housing of our equipment(eth0 eth1)

Reply:
Great, then just label all ports to match the silkscreen instead of mix and matching

Reply:
Since you pointed out that there is silkscreen, name both ports according to it